### DISCUSSION TITLE
Away for codefreeze ❄️ until Jan 12

### DISCUSSION THREAD

**Discussion by mrcjkb** [mrcjkb/rustaceanvim#121] [Announcements]
Hey everyone.

This week, until January 12, I will be attending [codefreeze](https://codefreeze.fi/) in Kilopää, Finland.

I will be taking my laptop, and hope to be able to respond to issues asap. But I may not be as quick as you are used to from me.

Cheers :smile:

**Comment by roland-5** [comment]
Take your time. Have a nice trip.
